modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/legacy/upload.js

```diff
@@ -1164,6 +1166,8 @@ AUI.add(
 						});
 
 						Liferay.Util.openToast({
+							autoClose:
+								invalidFilesLength > 3 ? undefined : false,
```

Comment:
What means the "undefined" here?

Reply:
I use `undefined` to get the [default value](https://github.com/liferay/liferay-portal/blob/39a09a4346ea519c076c1828a5a0dda4a67c4ecb/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/toast/commands/OpenToast.es.js#L100) 5000ms in this case.

Reply:
Wouldn't be better to pass `true` value, and then internally this time is set to 10000ms?  You are also setting this value [here](https://github.com/liferay-lima/liferay-portal/pull/3095/files#diff-6834d19ec86c87a0c1ae24099a0b076daa3a28f0c11678428c43a07a5fce71d1R1103). If not, I would prefer to pass `5000` ms as parameter (it is clear for the developer) instead of `undefined`, which does not indicate what will do the component do. What do you think? I would leave it as 							`autoClose: invalidFilesLength > 3`

Reply:
Another option, if you really want to set it to 5000ms (anyway I'll pass this value as `autoClose: 5000 || false`), but you prefer to get the default value, could be something like this: 

```
const baseProps = {
       message,
       toastProps: {
	     className: 'alert-full',
       },
       type: 'danger'
}

if (invalidFilesLength > 3) {
      baseProps = {...baseProps, autoClose: false}
}
```


For me this is more clear than passing an `undefined` value.